Assembly reqs

maximilianpress opened the pull request:
initial draft of assembly guidelines for PG docs. a couple of to-be-continueds, but probably better to have live documentation even if there are gaps.
would like to get input from other members of PG team, too.

Commit:
outlined assembly recommendations, needs fleshing out

Commit:
some more stuff, added chimera image

Commit:
added more text about problems, and images as examples

Commit:
added some text about fixing problems in assemblies

Commit:
added section about contiguity guidelines.

Commit:
added section about contiguity guidelines.

Commit:
added a couple of things before pull request for adding assembly guidelines.

bnelsj commented:
I think in general the requirements looks good, but I think some of the wording could use a bit of work, or at least some more discussion.

1. Since it seems like most of the assemblies we scaffold fall into the *not encouraged* category, should we be worried about losing business or upsetting current customers? I'm worried this statement might be too strong.
2. There are a few places where we talk about choice of technology making things difficult to scaffold. I think it would be better to rephrase this in terms of the final result (worse on average).
3. It seems a bit inconsistent to put 10X in the "not encouraged" category in section 1 and the "strongly recommended" category in section 3. Additionally, do we think the technologies in 3 are necessary if the customer has 70X PacBio? A couple possible resolutions here would be to add an "encouraged" category in section 1 for 10X + Illumina assemblies, or reduce the strength of the recommendation in section 3.

Commit:
responding to comments - encouraged/discouraged

Commit:
more edits in response to review comments

maximilianpress commented:
in response to Brad's comments:

1. changed wording from "discouraged" to "acceptable in some cases".
2. changed wording from "difficult to scaffold" to talking about final results.
3. 10X is now "acceptable in some cases" for generating a starting assembly, so less inconsistent. I think it's also worth noting that the complementary technologies should be presented as a way to evaluate the starting assembly and make minor improvements, e.g. contig breaking, rather than in generating a starting assembly. If there's a way to be clearer about this then suggestions would be great.

It's worth noting that we are even now getting a customer who had PacBio data to go back and get 10X data as a way to improve their starting assembly. Certainly the final results that have been very successful, e.g. goat, tend to combine several approaches for starting scaffolds and then use Hi-C to create chromosome-level superscaffolds.

Commit:
some final changes before merge.
